_config.yml

```diff
@@ -116,126 +144,146 @@ stable_releases:
           url: "https://www.apache.org/dyn/closer.lua/flink/flink-1.7.2/flink-1.7.2-bin-scala_2.12.tgz"
           asc_url: "https://www.apache.org/dist/flink/flink-1.7.2/flink-1.7.2-bin-scala_2.12.tgz.asc"
           sha512_url: "https://www.apache.org/dist/flink/flink-1.7.2/flink-1.7.2-bin-scala_2.12.tgz.sha512"
+      optional_components:
+        -
+          name: "Avro SQL Format"
+          category: "SQL Formats"
+          scala_dependent: false
+          id: 172-sql-format-avro
+          url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-avro/1.7.2/flink-avro-1.7.2.jar
+          asc_url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-avro/1.7.2/flink-avro-1.7.2.jar.asc
+          sha_url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-avro/1.7.2/flink-avro-1.7.2.jar.sha1
+        -
+          name: "JSON SQL Format"
+          category: "SQL Formats"
+          scala_dependent: false
+          id: 172-sql-format-json
+          url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-json/1.7.2/flink-json-1.7.2.jar
+          asc_url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-json/1.7.2/flink-json-1.7.2.jar.asc
+          sha_url: https://repository.apache.org/content/groups/public/org/apache/flink/flink-json/1.7.2/flink-json-1.7.2.jar.sha1
+      alternative_binaries: 
```

Comment:
This reorganization should be done in a separate commit and deserves it's own discussion.

Reply:
separate PR now